frontend/README.md

```diff
@@ -0,0 +1,59 @@
+# Knative Showcase Frondend written in React/Typescript
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app)
+with the `typescript` template.
+
+## Available Scripts and Targets
+
+This project have both `make` and `npm` build scripts. In the project directory,
+you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
```

Comment:
Nit: Are the backslashes at the end of the line necessary? I'm seeing them for the first time in an `.md` file. I'd probably leave line-wrapping to the browser (same on a few line below). Anyway, many lines in this file are longer (without backslash).

Reply:
Done in ced694a2789d20055a2a6534340e0c945f1387ce